lib/src/main/java/com/metaplex/lib/experimental/jen/candyguard/idl.kt

```diff
@@ -9,7 +9,7 @@ package com.metaplex.lib.experimental.jen.candyguard
 
 val candyGuardJson = """
```

Comment:
We need to start removing this IDLs form code an use a reliable source

Reply:
agreed